Sources/PackageDescription/Package.swift

```diff
@@ -217,6 +217,10 @@ func manifestToJSON(_ package: Package) -> String {
     return JSON.dictionary(dict).toString()
 }
 
+public func jsonString(package: Package) -> String {
+    return package.toJSON().toString()
+}
```

Comment:
Let's leave a FIXME here or some other doc explaining why this is a free public function versus the rest of the code just using `PackageDescription.toJSON`...

I'm beginning to think we should move to mirroring the `PackageDescription` types inside `PackageModel` sooner rather than later, and stop explicitly importing `PackageDescription` anywhere.